src/services/travisauth.service.js

```diff
@@ -55,28 +60,66 @@ export default class TravisAuthenticationService {
     return null;
   }
 
+
+  /**
+   * Exchanges a github token for a travis access token. This uses Travis API V2, which will be
+   * deprecated eventually.
+   *
+   * @param githubToken the github token to exchange
+   * @returns {Promise<string | *>} the travis access token
+   */
+  static async exchangeGithubTokenForTravisToken(githubToken) {
+    const url = 'https://api.travis-ci.org/auth/github';
+
+    try {
+      const response = await fetch(url, {
+        method: 'POST',
+        headers: {
+          'User-Agent': 'Travis/1.0',
+          Accept: 'application/vnd.travis-ci.2+json',
```

Comment:
Does Accept need to be in single quotes as well?

Reply:
Nope. If it is a one word without any funny characters, it don't need quotes. linter will catch this